doc/sphinx/source/installation/software/linux.rst

```diff
@@ -4,4 +4,125 @@
 Linux
 ======================
 
-TBD
+Paparazzi is very easily installed on any laptop or workstation running Ubuntu, Debian (or any of their derivatives).
+
+The steps required to install the software needed to be able to let your UAS fly
+
+* Install the basic Paparazzi dependencies and the ARM cross compiling toolchain
+* Download the source code from the source repository
+* Allow access to your PC hardware connection by adding appropriate Udev rules
+* Compile the binaries from the sources and launch the software
+
+Users of other Linux flavors than a recent Ubuntu or Debian and anyone needing manual control of each individual package can install them independently.
+
+For the impatient
+=================
+
+For Ubuntu add the `paparazzi-uav ppa <https://launchpad.net/~paparazzi-uav/+archive/ubuntu/ppa>`_ (*sudo add-apt-repository ppa:paparazzi-uav/ppa*)
+and install the paparazzi-dev package.
+
+
+.. raw:: html
+
+    <iframe width="425" height="239" src="https://www.youtube.com/embed/SshFJrBuku8" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+
+
+Installation of dependencies
+============================
+______
+Ubuntu
+______
+
+Binary packages for Ubuntu are available for the i386, amd64 and armhf architectures.
+
+Add the installation sources for the Paparazzi software packages. Run from a terminal:
+
+::
+
+    sudo add-apt-repository ppa:paparazzi-uav/ppa
+
+Then update the systems package inventory and install the main Paparazzi software dependencies. This will take some time.
+
+::
+
+    sudo apt-get update 
+    sudo apt-get install paparazzi-dev
+
+______
+Debian
+______
+
+Binary packages for Debian are available for the i386 and amd64 architectures. armhf packages seem to be currently not supported by the OpenSUSE build service.
+
+For Debian Wheezy (7.0), Jessie (8.0) and Stretch (9.0) packages are built using the Open Build Service (OBS) on OpenSUSE Build Service project home:flixr:paparazzi-uav
```

Comment:
actually, we also have Debian10, but the lablgtk package is broken (same issue than with Ubuntu 20.04, but we can't use the fixed package). This might be repaired some day...

Reply:
I plan on changing a bit this part of the doc. Maybe in a future PR ? I added a warning on the top of the page.